src/mui/field/RichTextField.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+
+export const stripTags = input => input.replace(/<[^>]+>/gm, '');
+
+const RichTextField = ({ source, record = {}, stripped }) => {
```

Comment:
I suggest renaming `stripped` to `stripTags`, and setting it to `false` by default